test/e2e/accounts/smart-swap-disabled.spec.ts

```diff
@@ -1,17 +1,19 @@
-import { title } from 'process';
 import { Suite } from 'mocha';
-import { withFixtures } from '../helpers';
+import { withFixtures, defaultGanacheOptions } from '../helpers';
 import { Driver } from '../webdriver/driver';
-import {
-  accountSnapFixtures,
-  installSnapSimpleKeyring,
-  makeNewAccountAndSwitch,
-} from './common';
+import { installSnapSimpleKeyring, makeNewAccountAndSwitch } from './common';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+const FixtureBuilder = require('../fixture-builder');
```

Comment:
Any reason on why you can't just import using the required syntax?

```suggestion
import FixtureBuilder from '../fixture-builder';
```

Reply:
No? Good question/suggestion.  Updated.  Thanks